BUILD_WIN.md

```diff
@@ -53,22 +53,41 @@ Download the file named win64-x64 Installer from the [CMake Website](https://cma
 
 Install version 10.15.0 LTS (or greater) of [Node.JS and NPM](<https://nodejs.org/en/download/>).
 
-### Step 5. Create VCPKG environment variable
+### Step 5. Qt
+
+Install version 5.12.3 of [Qt](<https://www.qt.io/download-open-source>), as well as the following packages:
```

Comment:
```suggestion
If you would like to compile Qt instead of using the precompiled package provided during CMake, you can do so now. Install version 5.12.3 of [Qt](<https://www.qt.io/download-open-source>), as well as the following packages:
```